lib/Runtime/Debug/DiagObjectModel.cpp

```diff
@@ -2197,9 +2186,10 @@ namespace Js
 
 #if ENABLE_TTD
         bool suppressGetterForTTDebug = scriptContext->GetThreadContext()->IsRuntimeInTTDMode() && scriptContext->GetThreadContext()->TTDLog->ShouldDoGetterInvocationSupression();
+        TTD::TTModeStackAutoPopper suppressModeAutoPopper(scriptContext->GetThreadContext()->TTDLog);
         if(suppressGetterForTTDebug)
         {
-            instance->GetScriptContext()->GetThreadContext()->TTDLog->PushMode(TTD::TTDMode::DebuggerSuppressGetter);
+            suppressModeAutoPopper.PushModeAndSetToAutoPop(TTD::TTDMode::DebuggerSuppressGetter);
```

Comment:
>suppressModeAutoPopper [](start = 12, length = 22)

This change was not in node PR [https://github.com/nodejs/node-chakracore/pull/158](https://github.com/nodejs/node-chakracore/pull/158), do we need this as well?